test/integration/register-user.test.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const request = require("supertest");
+import routes from "../../src/routes/index";
+import UserModel from '../../src/models/users'
+const app = express();
```

Comment:
Since this is the same for the integration tests, lets separate it out so it can be reused in both.

Reply:
I created the import setup on `src/utils/integrationTestSetup.js` to reuse base setup.